src/celengine/globular.cpp

```diff
@@ -492,7 +492,8 @@ void Globular::renderGlobularPointSprites(
     globProg->use();
 
     globularTex->bind();
-    globProg->mat4Param("MVPMatrix") = mvp;
+    globProg->ProjectionMatrix = *m.projection;
+    globProg->ModelViewMatrix = *m.modelview;
     globProg->mat4Param("ModelViewMatrix") = vecgl::translate(*m.modelview, offset);
```

Comment:
and here... MVPMatrix is not a multiply of Projection * ModelView

Reply:
yes, i remember i stuck with this when was converting Celestia to GL ES, but i left this for future fix. i need to check the original code. but again, currently you can set all 3 matrices independently and we will fix this later.